### PR TITLE
FORMS-14275 Embed config override tests apply

### PR DESCRIPTION
**Pull request by rajatofficial**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.tests/test-module/specs/embedFormsInSiteOverrideConfig/embedFormsInSiteOverrideConfig.spec.js [hunk 1]
```diff
@@ -33,22 +33,22 @@ describe("Form in site using embed container", () => {
...
       });
 
-      it.skip("should submit on button click and thank you message of embed container must be overridden", () => {
+      it("should submit on button click and thank you message of embed container must be overridden", () => {
             cy.intercept({
             method: 'POST',
             url: '**/adobe/forms/af/submit/*',
         }).as('afSubmission');
-        cy.get(`.cmp-adaptiveform-button__widget`).eq(0).should('be.visible').click().then(() => {
+        cy.get(`.cmp-adaptiveform-button__widget`).eq(0).should('be.visible').wait(500).click().then(() => {
```
Comment (rismehta) [inline ui.tests/test-module/specs/embedFormsInSiteOverrideConfig/embedFormsInSiteOverrideConfig.spec.js:41]: Lets not add wait in the test case, it gets flaky with time, wait for an element instead

File: ui.tests/test-module/specs/embedFormsInSiteOverrideConfig/embedFormsInSiteOverrideConfig.spec.js [hunk 2]
```diff
@@ -33,22 +33,22 @@ describe("Form in site using embed container", () => {
         expect(formContainer.length).to.equal(2);
       });
 
-      it.skip("should submit on button click and thank you message of embed container must be overridden", () => {
+      it("should submit on button click and thank you message of embed container must be overridden", () => {
```
Comment (rismehta) [inline ui.tests/test-module/specs/embedFormsInSiteOverrideConfig/embedFormsInSiteOverrideConfig.spec.js:36]: Both these test should only run in latestAddon. can you add the cy.af.isLatestAddon check ? you can check for the occurrences

**Review by rismehta — CHANGES_REQUESTED** [review]
check comments

File: ui.tests/test-module/specs/embedFormsInSiteOverrideConfig/embedFormsInSiteOverrideConfig.spec.js (hunk 2 above)
Comment (rajatofficial) [reply ui.tests/test-module/specs/embedFormsInSiteOverrideConfig/embedFormsInSiteOverrideConfig.spec.js]: Added

File: ui.tests/test-module/specs/embedFormsInSiteOverrideConfig/embedFormsInSiteOverrideConfig.spec.js (hunk 1 above)
Comment (rajatofficial) [reply ui.tests/test-module/specs/embedFormsInSiteOverrideConfig/embedFormsInSiteOverrideConfig.spec.js]: Removed wait.

**Review by rismehta — APPROVED** [review]

**Pull request merged**